pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -3445,6 +3465,68 @@ private void internalExpireMessagesByTimestampForSinglePartition(String subName,
         }
     }
 
+    private CompletableFuture<Void> internalExpireMessagesByTimestampForSinglePartitionAsync(
```

Comment:
`internalExpireMessagesByTimestampForSinglePartition` should be implemented by this async method.

Reply:
The method is currently available only use in internalExpireMessagesForAllSubscriptionsForNonPartitionedTopic，the follow-up will be asynchronous, and delete sync method.